examples/blog/src/components/BlogPostPreview.astro

```diff
@@ -1,48 +1,36 @@
 ---
 export interface Props {
-	post: any;
+	title: string;
+	description: string;
+	publishDate: string;
+	url: string;
 }
 
-const { post } = Astro.props;
+const { title, description, publishDate, url } = Astro.props as Props;
 ---
 
 <article class="post-preview">
 	<header>
-		<p class="publish-date">{post.frontmatter.publishDate}</p>
-		<a href={post.url}><h1 class="title">{post.frontmatter.title}</h1></a>
+		<p class="publish-date">{publishDate}</p>
```

Comment:
While we're at it, this should probably be a `<time>` element.  That'd also allow scoped styles without a class selector 👍

Reply:
Ah yes LOVE me a good `time`